vkubelet/node.go

```diff
@@ -169,27 +168,25 @@ func (n *Node) Run(ctx context.Context) error {
 		return err
 	}
 
-	if !n.disableLease {
-		n.lease = newLease(n.lease)
-		setLeaseAttrs(n.lease, n.n, n.pingInterval*5)
-
-		l, err := ensureLease(ctx, n.leases, n.lease)
-		if err != nil {
-			if errors.IsNotFound(err) {
-				n.disableLease = true
-			} else {
-				return pkgerrors.Wrap(err, "error creating node lease")
-			}
-		}
-		log.G(ctx).Debug("Created node lease")
-
-		n.lease = l
+	if n.leases == nil {
+		n.disableLease = true
+		return n.controlLoop(ctx)
 	}
 
-	if n.disableLease {
+	n.lease = newLease(n.lease)
+	setLeaseAttrs(n.lease, n.n, n.pingInterval*5)
```

Comment:
Why `pingInterval*5`?

Reply:
This makes sure the lease is valid for plenty of time.
I just picked what seemed like something good enough since the ping interval is typically very small and I want to make sure the lease doesn't go invalid because of a temporary problem.

Can update if you think of a better value.

Reply:
We can update this as needed (or maybe provide an option override this). This value is what was originally set (of course did double the default ping interval here).